internal/deviations/deviations.go

```diff
@@ -224,4 +224,6 @@ var (
 		"Device skip isis restart-suppress check if value is true, Default value is false")
 
 	MacAddressMissing = flag.Bool("deviation_mac_address_missing", false, "Device does not support /system/mac-address/state.")
+
+        SchedulerInputWeightLimit = flag.Bool("deviation_scheduler_input_weight_limit", false, "Device does not support weight above certain limit")
```

Comment:
Please follow the new style for any new deviations. The flag should be unexported and accessed through an exported accessor method that accepts a DUT, e.g.
https://github.com/openconfig/featureprofiles/blob/main/internal/deviations/deviations.go#L73
This is going to enable our transition away from flags to reading vendor-specific deviations from the metadata textproto.

Sample PR to see the deviation and use in test - https://github.com/openconfig/featureprofiles/pull/1358/files